docs/GPS.md

```diff
@@ -1,5 +1,41 @@
 #Phidget GPS Sensor
 
-[Basic GPS example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/GPS.js)
+The PhidgetGPS library makes for intuitive and lightning fast development without any compromise. For a quick start into your GPS project see the [basic GPS example.](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/GPS.js).  
 
-ToDo : map out module and describe use
\ No newline at end of file
+
+##Getting Started
+
+Initializing [Phidgets GPS Devices](http://www.phidgets.com/products.php?product_id=1040) will require:
+
+    var Phidget = require('phidgeapi.js').GPS;
+
+along with a variable name:
+
+    var GPS = new Phidget;
```

Comment:
You do not have to do it this way for example :

var GPS=new require('phidgeapi.js').GPS;

So we probably should not call this out.